test/libsolidity/util/TestFileParser.cpp

```diff
@@ -85,6 +85,21 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						call.kind = FunctionCall::Kind::Library;
 						call.expectations.failure = false;
 					}
+					else if (accept(Token::Storage, true))
+					{
+						expect(Token::Colon);
+						call.expectations.failure = false;
+						call.expectations.result.push_back(Parameter());
+						// empty / non-empty is encoded as false / true
```

Comment:
Would it be more intuitive if `empty` is encoded to `true`?

Reply:
I thought it makes sense that `0` is encoded as empty... ("set to zero")